### PR TITLE
Foodcritic fixes

### PR DESCRIPTION
**Pull request by iankronquist**
This will fix some of the Travis CI issues.

### CODE REVIEW

**Comment by iankronquist** [comment]
@bramwelt Since jordan is out of town, can you take a look at this?
@tschuy r?

File: providers/gunicorn.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@
       base_command = "#{::File.join(django_resource.virtualenv, "bin", "python")} manage.py run_gunicorn"
     else
       gunicorn_command = new_resource.virtualenv.nil? ? "gunicorn" : "#{::File.join(new_resource.virtualenv, "bin", "gunicorn")}"
-      base_command = "#{gunicorn_command} #{new_resource.app_module}"
+      base_command = #{gunicorn_command} #{new_resource.app_module}
```
Comment (armiller) [inline providers/gunicorn.rb]: Is this valid ruby?

**Comment by armiller** [comment]
@iankronquist I will look at this in more detail tomorrow.

File: providers/gunicorn.rb (hunk 1 above)
Comment (jordane) [inline providers/gunicorn.rb]: Technically yes, but it isn't what you actually want to do.

The `=` carries on to the following line.

**Comment by iankronquist** [comment]
See issue #13

**Pull request closed without merging**